src/models/Detour.js

```diff
@@ -0,0 +1,21 @@
+const { DataTypes, UUIDV4 } = require('sequelize')
+
+// Create & export Role model
+module.exports = (sequelize) => {
+  sequelize.define('Detour', {
+    id: {
+      type: DataTypes.UUID,
+      defaultValue: UUIDV4,
+      primaryKey: true
+    },
+    amount: {
+      type: DataTypes.DOUBLE,
+      allowNull: false
+    },
+    observation: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+  },
+  { timestamps: false });
```

Comment:
Now, I think we need it.

Reply:
- I set the allowNull to false so that the detour has to be described. Should it be disabled?
- Disable timeStamp to avoid filling with unnecessary data for the moment. If necessary we enable it.